applications/nrf_desktop/configuration/common/led_state.h

```diff
@@ -47,6 +47,17 @@ enum led_id {
 
 #define LED_UNAVAILABLE CONFIG_DESKTOP_LED_COUNT
 
+#if (defined(CONFIG_BT_PERIPHERAL) && defined(CONFIG_BT_CENTRAL)) || \
+    ((!defined(CONFIG_BT_PERIPHERAL)) && (!defined(CONFIG_BT_CENTRAL)))
```

Comment:
some brackets are excessive and do not add to overall readability

Reply:
ok, removed